### PR TITLE
updated network restriction documentation

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 1]
```diff
@@ -125,12 +125,19 @@ In most cases, if you have thousands of users and entities, `10GB` of disk space
 
 ## Network Access
 
-FusionAuth downloads the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. After that version, they are only downloaded if you are using MySQL.
+FusionAuth offers instructions on downloading the MySQL JDBC driver at install time. Due to the drivers' licensing, it cannot be bundled into the application. Prior to 1.16.0, the drivers were downloaded no matter which database you used. Prior to 1.40.0, the drivers were downloaded if you are using MySQL.
```
Comment (spwitt) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:128]: This means that in `1.40.0` and later they must download and install the drivers even if the server has network access. Should we call that out?

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 2]
```diff
@@ -125,12 +125,19 @@ In most cases, if you have thousands of users and entities, `10GB` of disk space
...
 
 If you are running FusionAuth in an environment with no network access and are using MySQL, you must download the MySQL driver jar file and place it in `fusionauth-app/web/WEB-INF/lib`.
 
 ### Paid Editions
 
-If you are using a paid edition of FusionAuth, the instance must have outbound network connectivity for license id verification and metrics reporting. You must also allow access to enable advanced features provided by FusionAuth Reactor cloud services.
+If you are using the community plan, you can block all outbound network connectivity. Lack of connectivity will not affect the product. 
+
+Once on a paid plan, you must enable network access to FusionAuth or your license will not function properly. You must also allow access to enable advanced features provided by FusionAuth Reactor cloud services.
```
Comment (spwitt) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:136]: The "you must enable" and "you must also" sounded too demanding, but it's up to you if you want to change it.
```suggestion
Once on a paid plan, you must enable network access to FusionAuth, or your license will not function properly. Advanced features provided by FusionAuth Reactor cloud services require additional outbound network access.
```

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx [hunk 3]
```diff
@@ -125,12 +125,19 @@ In most cases, if you have thousands of users and entities, `10GB` of disk space
...
+
+If you are in a firewalled or restricted environment and have a paid plan, add the following hostnames to your network allow list:
```
Comment (spwitt) [inline astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:138]: Should we call out how each of these is used?

**Review by spwitt — CHANGES_REQUESTED** [review]

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx (hunk 1 above)
Comment (mooreds) [reply astro/src/content/docs/get-started/download-and-install/system-requirements.mdx]: Good idea, done.

File: astro/src/content/docs/get-started/download-and-install/system-requirements.mdx (hunk 3 above)
Comment (mooreds) [reply astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:140]: I don't see any reason to do this. If you want to use a paid edition, you need to enable access to all three, what does it matter why?

Is there some reason I'm missing?

Comment (spwitt) [reply astro/src/content/docs/get-started/download-and-install/system-requirements.mdx:140]: I just assume that someone is going to ask what they are used for now that the specific domains are documented. I don't think it's strictly necessary and how each is used is subject to change.

**Pull request merged**